shared/src/types/admin/Payment.ts

```diff
@@ -17,12 +18,14 @@ export type Payment = {
 	payment_at: Timestamp;
 	status: PaymentStatus;
 	comments?: string;
+	message?: EntityReference[];
```

Comment:
since it's an array: `messages`?

Reply:
I think it is required like that to make a reference to the message, isn't it?